prereise/gather/demanddata/transportation_electrification/immediate.py

```diff
@@ -0,0 +1,184 @@
+# DELETED: comm_type, LOAD, vmtsc
+
+import prereise/gather/demanddata/transportation_electrification/mileage
+import prereise/gather/demanddata/transportation_electrification/charging
+import numpy as np
+
+
+def immediate_charging(
+    census_region, model_year, veh_range, kwhmi, power, location_strategy
+):
+    """Immediate charging function
+
+    :param int census_region: any of the 9 census regions defined by US census bureau.
+    :param int model_year: year that is being modelled/projected to, 2017, 2030, 2040, 2050.
+    :param int veh_range: 100, 200, or 300, represents how far vehicle can travel on single charge.
+    :param int kwhmi: fuel efficiency, should vary based on vehicle type and model_year.
+    :param int power: charger power, EVSE kW.
+    :param int location_strategy: where the vehicle can charge-1, 2, 3, 4, or 5; 1-home only, 2-home and work related, 3-anywhere if possibile, 4-home and school only, 5-home and work and school.
+    :return: (*numpy.ndarray*) -- charging profiles.
+    """
+    # load NHTS data from function
+    newdata = mileage.remove_ldt(mileage.load_data(census_region))
+    # add new columns to newdata to store data that is not in NHTS data
+    newdata['trip start battery charge'] = 0
+    newdata['trip end battery charge'] = 0
+    newdata['charging power'] = 0
+    newdata['charging time'] = 0
+    # jx: never actually used... delete?
+    newdata['charging consumption'] = 0  # needs double check
+    newdata['BEV could be used'] = 0
+    newdata['trip number'] = 0
+
+    input_day = mileage.get_input_day(mileage.get_model_year_dti(model_year))
+
+    TRANS_charge = np.zeros(24 * len(input_day))
+
+    data_day = mileage.get_data_day(newdata)
+
+    actual_vmt = 0
+    potential_vmt = 0
+
+    # 1-anytrip number, 2-last trip
+    trip_strategy = 1
+
+    kwh = kwhmi * veh_range
+
+    for day_iter in range(len(input_day)):
+
+        electricload = np.zeros(4800)
+        # initializes electricload_old
+
+        # vmt, trip amount, vehicle amount
+        info1 = [0] * 3

```

Comment:
What is the purpose of this list? It seems that the second and third entries get modified, but never actually used.

Reply:
Right now we only use info1[0] (vmt) because it's used for the output. But you'll notice that we use it in a commented out section. This is just for Kate and I to test the code.

>  

        # only used as "test" output variables
        # output_load.append(outputelectricload)
        # info_out.append(info1)